bower.json

```diff
@@ -9,9 +9,22 @@
     "testing"
   ],
   "devDependencies": {
-    "web-component-tester": "^4.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
-    "webcomponentsjs-v1": "webcomponents/webcomponentsjs#v1"
+    "web-component-tester": "^4.0.0 || 6.0.0-prerelease.5",
```

Comment:
Yeah, I think we're going to end up with the same problem here. Changed it to `^6.0.0-prerelease.5`.